src/sentry/filestore/gcs.py

```diff
@@ -16,7 +16,7 @@
 from google.cloud.storage.blob import Blob
 from google.cloud.storage.bucket import Bucket
 from google.cloud.exceptions import NotFound
-from google.auth.exceptions import TransportError
+from google.auth.exceptions import GoogleAuthError
```

Comment:
`GoogleAuthError` covers `TransportError`, `RefreshError`, and `DefaultCredentialsError`

Reply:
Actually, let's exclude`DefaultCredentialsError`.